packages/welcome-screen/src/views/PermissionsWrongFolder.vue

```diff
@@ -12,7 +12,7 @@
       </h5>
       <b-button
         variant="primary"
-        @click="loadWithUSB"
+        @click="onClick"
```

Comment:
It would be nice if this click handler had a more specific name, since there are multiple components in this tree which could be clicked. Perhaps `onNextButtonClick`?

Reply:
You can see it was like that before: the handler was called "loadWithUSB". But this is the only click handler in this component so I think because of scoping is fine to call it just `onClick`.